frontend/src/concepts/projects/ProjectsContext.tsx

```diff
@@ -44,25 +46,29 @@ type ProjectsProviderProps = {
 };
 
 const ProjectsContextProvider: React.FC<ProjectsProviderProps> = ({ children }) => {
-  const fetchProjects = React.useCallback(() => getDSGProjects(), []);
+  const fetchProjects = React.useCallback(() => getProjects(), []);
   const [preferredProject, setPreferredProject] =
     React.useState<ProjectsContext['preferredProject']>(null);
   const [projectData, loaded, loadError, refreshProjects] = useFetchState<ProjectKind[]>(
     fetchProjects,
     [],
   );
 
-  const { projects, modelServingProjects, nonActiveProjects } = React.useMemo(
+  const { projects, dataScienceProjects, modelServingProjects, nonActiveProjects } = React.useMemo(
     () =>
       projectData.reduce<{
         projects: ProjectKind[];
+        dataScienceProjects: ProjectKind[];
         modelServingProjects: ProjectKind[];
         nonActiveProjects: ProjectKind[];
       }>(
         (states, project) => {
           if (project.status?.phase === 'Active') {
             // Project that is active
             states.projects.push(project);
+            if (project.metadata.labels?.[KnownLabels.DASHBOARD_RESOURCE]) {
+              states.dataScienceProjects.push(project);
```

Comment:
Now that you're fetching everything -- you need to trim out `openshift-` and `kube-`, as well as remove `default` -- these are not valid projects.

Reply:
We had the discussion in the UX meeting last week, it's a short one and you could go back to check that, I think we came to the agreement at last we are not going to filter any projects. I also added it here in the comment https://github.com/opendatahub-io/odh-dashboard/pull/1643#issuecomment-1678941615

Reply:
I'm going to have to overrule that -- absolutely not a good idea. I'll watch the recording (still on my todo list) but we 100% do not want to expose that in our UI -- accidental misuse of this projects is a *bad* idea.